src/tests.rs

```diff
@@ -22,7 +22,7 @@ fn find_library_folders() {
     let mut steamdir = steamdir_found.unwrap();
 
     steamdir.libraryfolders.discover(&steamdir.path);
-    assert!(steamdir.libraryfolders().paths.len() > 1);
+    assert!(steamdir.libraryfolders().paths.len() > 0);
```

Comment:
The comment at the top of the file should be updated to drop the two required libraries bit

Reply:
initially this was added to makwe local testing easier, should this one line be reverted back to how it was originally?

Reply:
I'm fine with either as long as the comment is accurate. I'm planning on totally overhauling the testing as a part of v2, so that it doesn't require an actual installation